### PR TITLE
[s]Computer ID is a string, Thanks byond.

### PR DESCRIPTION
**Pull request by RemieRichards**
I chose text2num() over putting the integer max in quotes since I believe text2num() is nicer than doing a string comparison.

@MrStonedOne 
@Fox-McCloud 

> computer_id var (client)
> Default value: 
> A unique numerical identifier for the player's computer. The value is in text form.

### CODE REVIEW

**Comment by duncathan** [comment]
yknow we do have a security tag

**Comment by RemieRichards** [comment]
@duncathan I forget.

File: code/modules/admin/IsBanned.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		log_access("Failed Login (invalid data): [key] [address]-[computer_id]")
 		return list("reason"="invalid login data", "desc"="Error: Could not check ban status, Please try again. Error message: Your computer provided invalid or blank information to the server on connection (byond username, IP, and Computer ID.) Provided information for reference: Username:'[key]' IP:'[address]' Computer ID:'[computer_id]'. (If you continue to get this error, please restart byond or contact byond support.)")
 	
-	if (computer_id == 2147483647) //this cid causes stickybans to go haywire 
+	if (text2num(computer_id) == 2147483647) //this cid causes stickybans to go haywire
```
Comment (Menshin) [inline code/modules/admin/IsBanned.dm]: So, _computer_id_ (converted to int) cannot be over Integer max after all ?

Comment (RemieRichards) [inline code/modules/admin/IsBanned.dm]: It's weirder, it can't BE integer max, but it CAN be over it.

**Comment by GinjaNinja32** [comment]
`2147483647

**Comment by duncathan** [comment]
Who knows, maybe this one will fix it

**Comment by Fox-McCloud** [comment]
@RemieRichards ping me if this does, in fact, resolve the issue?

**Pull request merged**